modin/core/io/text/text_file_dispatcher.py

```diff
@@ -591,13 +591,26 @@ def _define_metadata(
 
         return column_widths, num_splits
 
+    _parse_func = None
+
+    @classmethod
+    def get_parse_func(cls):  # noqa: RT01
+        """Prepare a function for transmission to remote workers."""
+        if cls._parse_func is None:
+            cls._parse_func = cls.put(cls.parse)
+        return cls._parse_func
+
     @classmethod
-    def _launch_tasks(cls, splits: list, **partition_kwargs) -> Tuple[list, list, list]:
+    def _launch_tasks(
+        cls, *args, splits: list, **partition_kwargs
```

Comment:
```suggestion
        cls, splits: list, *partition_args, **partition_kwargs
```
I don't really like this order. Can we use as suggested?

Reply:
The current solution implies that this parameter must always be passed by name. For me, this is more explicit behavior. Why don't you like it?